app/presenters/content_item_presenter.rb

```diff
@@ -18,6 +18,12 @@ def available_translations
     sorted_locales(@content_item["links"]["available_translations"])
   end
 
+  def parent
+    if content_item["links"].include?("parent")
+      content_item["links"]["parent"][0]
```

Comment:
#parent is used for other things not related to breadcrumbs. So would leave it separate.